src/OrchardCore/OrchardCore.Infrastructure.Abstractions/MemoryStreamFactory.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.IO;
+
+namespace OrchardCore;
+
+public static class MemoryStreamFactory
+{
+    private static readonly RecyclableMemoryStreamManager _manager = new();
+
+    public static MemoryStream GetStream(byte[] buffer, int offset, int count, string tag = null)
+        => _manager.GetStream(tag, buffer, offset, count);
+
+    public static MemoryStream GetStream(byte[] buffer, string tag = null)
+        => _manager.GetStream(tag, buffer, 0, buffer.Length);
+
+    public static MemoryStream GetStream()
```

Comment:
I believe this is the only overload that could be useful. Unless we expect to expand other buffers, but that would confuse most devs.